src/js/containers/agencyV2/accountSpending/TableContainer.jsx

```diff
@@ -112,14 +112,13 @@ const TableContainer = (props) => {
         if (currentPage !== 1) {
             changeCurrentPage(1);
         }
-        else {
-            fetchSpendingByCategoryCallback();
-        }
-    }, [props.type, props.fy, props.agencyId, pageSize, sort, order, totalObligation]);
+    }, [props.type, props.agencyId, pageSize, sort, order, totalObligation]);
```

Comment:
When I switch tabs, change the page size, etc. _while on page 1_ the table no longer makes an API request with the new params.